app/feature/insurances/src/main/kotlin/com/hedvig/android/feature/insurances/insurance/presentation/InsuranceViewModel.kt

```diff
@@ -0,0 +1,22 @@
+package com.hedvig.android.feature.insurances
+
+import com.hedvig.android.feature.insurances.data.GetCrossSellsUseCase
+import com.hedvig.android.feature.insurances.data.GetInsuranceContractsUseCase
+import com.hedvig.android.feature.insurances.insurance.presentation.InsurancePresenter
+import com.hedvig.android.feature.insurances.insurance.presentation.InsuranceScreenEvent
+import com.hedvig.android.feature.insurances.insurance.presentation.InsuranceUiState
+import com.hedvig.android.molecule.android.MoleculeViewModel
+import com.hedvig.android.notification.badge.data.crosssell.card.CrossSellCardNotificationBadgeService
+
+internal class InsuranceViewModel(
+  private val getInsuranceContractsUseCase: GetInsuranceContractsUseCase,
```

Comment:
You can remove `private val` from all these

Reply:
Thanks! 🤩